output_http.go

```diff
@@ -321,5 +321,6 @@ func (c *HTTPClient) Send(data []byte) ([]byte, error) {
 	if c.config.TrackResponses {
 		return httputil.DumpResponse(resp, true)
 	}
+	_ = resp.Body.Close()
```

Comment:
```suggestion
if req.Body != nil {
	_ = resp.Body.Close()
}
```
to avoid panics when the request doesn't have a body!

Reply:
Maybe there is no need to check the request body, because the response body needs to be closed, not the request body. According to the description of the response body in the golang documentation:
> The http Client and Transport guarantee that Body is always non-nil, even on responses without a body or responses with a zero-length body. It is the caller's responsibility to close Body.

So I think the response body can be closed directly.

Reply:
@code4wt you missed one important context, I'm gonna paste it here!

>  // Body is the request's body.
>     //
>     // For client requests, a nil body means the request has no
>     // body, such as a GET request. The HTTP Client's Transport
>     // is responsible for calling the Close method.
>     //
>     // For server requests, the Request Body is always non-nil
>     // but will return EOF immediately when no body is present.
>     // The Server will close the request body. The ServeHTTP
>     // Handler does not need to.
>     //
>     // Body must allow Read to be called concurrently with Close.
>     // In particular, calling Close should unblock a Read waiting
>     // for input.
>     Body io.ReadCloser

Reply:
In this context, the **Body** is the **request's body**. 
> The HTTP Client's Transport is responsible for calling the Close method.

Its meaning should be that HTTP Client's Transport is responsible for closing the request`s body. the code of http client as follow：
<img src="https://user-images.githubusercontent.com/25877772/118647810-56336e80-b814-11eb-8fc8-48e5144f81d4.png" width="80%" height="80%">

There is a similar question on StackOverflow ->[ is resp.Body.Close() necessary if we don't read anything from the body?]( https://stackoverflow.com/questions/18598780/is-resp-body-close-necessary-if-we-dont-read-anything-from-the-body)

Reply:
Oh great, I missed that we were talking about response's body